molgenis/molgenis-emx2
# chore: add directory to demoset

Add the BBMRI-ERIC directory to the demoset (Trac issue 10794)

how to test:
```
export MOLGENIS_INCLUDE_DIRECTORY_DEMO=true
./gradlew run
```
The directory-demo schema should now exist.
